charts/csi-powerstore/templates/node.yaml

```diff
@@ -79,7 +85,17 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 
 ---
-
+# Role for Driver-specific Permissions in a Namespace
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-node
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups: ["coordination.k8s.io"]
+    resources: ["leases"]
+    verbs: ["get", "watch", "list", "delete", "update", "create"]
+---
```

Comment:
This Role has only 1 rule, which is required only when Resiliency (podmon) is enabled. 
Please move this condition out so that the RoleBinding and Role are created only if podmon is enabled.